addons/danger/functions/fnc_checkBody.sqf

```diff
@@ -39,7 +39,7 @@ if (count _body > 0) exitWith {
             ((_unit distance _bodyPos) < 0.6) || {_time < time} || {!alive _unit}
         },
         {
-            params ["_unit", "_bodyPos", "_time", "_body"];
+            params ["_unit", "", "", "_body"];
             if (alive _unit && {!isNil str _body} && {_unit distance _pos < 0.4}) then {
```

Comment:
Minor tweak. 1.4.2 has set the range to 0.6 to more consistently trigger the rearm action.